pyqir/src/python.rs

```diff
@@ -71,5 +73,32 @@ fn _native(_py: Python, m: &PyModule) -> PyResult<()> {
     m.add_function(wrap_pyfunction!(result_id, m)?)?;
     m.add_function(wrap_pyfunction!(result_type, m)?)?;
     m.add_function(wrap_pyfunction!(result, m)?)?;
+
+    m.add_wrapped(wrap_pymodule!(_qis))?;
+    let sys = PyModule::import(py, "sys")?;
+    let sys_modules: &PyDict = sys.getattr("modules")?.downcast()?;
+    sys_modules.set_item("pyqir.qis._native", m.getattr("_qis")?)?;
+
+    m.add_wrapped(wrap_pymodule!(_rt))?;
+    let sys = PyModule::import(py, "sys")?;
+    let sys_modules: &PyDict = sys.getattr("modules")?.downcast()?;
+    sys_modules.set_item("pyqir.rt._native", m.getattr("_rt")?)?;
```

Comment:
This is really hacky... It'd be easier not to deal with the submodules in pyo3, you can add everything to pyqir._native, but re-export them from qis.py and rt.py instead of \_\_init\_\_.py.